paymentsheet/src/main/java/com/stripe/android/paymentsheet/injection/PaymentSheetCommonModule.kt

```diff
@@ -56,8 +56,8 @@ internal abstract class PaymentSheetCommonModule {
 
         @Provides
         @Named(PUBLISHABLE_KEY)
-        fun providePublishableKey(paymentConfiguration: Lazy<PaymentConfiguration>): () -> String =
-            { paymentConfiguration.get().publishableKey }
+        fun providePublishableKey(appContext: Context): () -> String =
+            { PaymentConfiguration.getInstance(appContext).publishableKey }
```

Comment:
This is so that if the currency changes after PaymentSheet is created, that the publishable key returned from the reload is updated in the PaymentSheet Configuration.